CONTRIBUTING.md

```diff
@@ -59,9 +59,10 @@ by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
     - Follow the [steps](https://help.github.com/en/articles/creating-a-pull-request) of the github help to create the PR.
     - Please note that your PR should be directed from your branch (for example `myfeature`) towards the branch `dev`
     - To make reviewing easier, briefly describe the changes you have made in the pull request and summarise the discussion and conclusions in the associated issue. 
+    - Write the corresponding issue number in the pull request so that they are linked. Write it with one of the [special keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) so that the issue will be automatically closed when the PR is merged (example: `Closes #<your issue number>`)
 7. Describe briefly (i.e. in one or two lines) what you changed in the `CHANGELOG.md` file. End the description by the number in parenthesis `(#<your PR number>)`
 8. Commit the changes to the `CHANGELOG.md` file
-9. Write the corresponding issue number in the pull request so that they are linked. Write it with one of the [special keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) so that the issue will be automatically closed when the PR is merged (example: `Closes #<your issue number>`)
+9. Add [term tracker items](https://github.com/OpenEnergyPlatform/ontology/wiki/term-tracker-item) to the main changed classes of the ontology
```

Comment:
Is it intentional that you replaced the number 9 with a new text? To it should be an additional point as I think the old number 9 is still valid.

Reply:
I moved number 9 up and made it a bullet point in writing a pull request

Reply:
there's no reason for it to be done that late

Reply:
Somehow I missed it.